_monitoring-plugins/alerting/index.md

```diff
@@ -11,7 +11,11 @@ redirect_from:
 OpenSearch Dashboards
 {: .label .label-yellow :}
 
-The alerting feature notifies you when data from one or more OpenSearch indices meets certain conditions. For example, you might want to notify a [Slack](https://slack.com/) channel if your application logs more than five HTTP 503 errors in one hour, or you might want to page a developer if no new documents have been indexed in the past 20 minutes.
+OpenSearch Dashboards provides the Alerting plugin that allows you to monitor your data and create notifications that trigger when conditions occur in one or more indexes.
```

Comment:
Can we rephrase this to "You can use the Alerting plugin in OpenSearch Dashboards to monitor your data and create notifications..."?